Src/AutoFixture.xUnit.net.UnitTest/DelegatingFixture.cs

```diff
@@ -101,5 +102,7 @@ public object Create(object request, ISpecimenContext context)
         }
 
         internal Func<object, ISpecimenContext, object> OnCreate { get; set; }
+
+        internal Action<ICustomization> OnCustomize { get; set; }

```

Comment:
Various Glue Libraries (_AutoFixture.AutoMoq_, _AutoFixture.AutoNSubstitute_, etcetera) each rely on their respective dynamic mock libraries (_Moq_, _NSubstitute_, etcetera), so I was concerned that using one of those in other parts would create dependency conflicts. Perhaps it was overkill...

We use _Moq_ in the _AutoFixture.AutoMoq_ codebase, _FakeItEasy_ in the _AutoFixture.AutoFakeItEasy_ code base, and so on.